src/actions/chatActions.js

```diff
@@ -0,0 +1,19 @@
+import { createThunk } from '@rootstrap/redux-tools';
+
+import chatService from 'services/chatService';
+import parseError from 'utils/parseError';
+
+const GET_CONVERSATIONS = 'GET_CONVERSATIONS';
+
+export const getConversations = createThunk(GET_CONVERSATIONS, async () => {
+  try {
+    const {
+      data: { matches },
+    } = await chatService.getConversations();
+    return matches;
```

Comment:
Another version of this could be:
```suggestion
  return (await chatService.getConversations()).data.matches
```